components/automate-ui/src/app/pages/notification-details/notification-details.component.html

```diff
@@ -0,0 +1,103 @@
+        <table>
+          <thead>
+            <tr class="detail-row">
+              <th class="header-column">Webhook Type</th>
+              <th class="header-column">Failure Type</th>
+              <th class="header-column">Webhook URL</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr class="detail-row">
+              <td *ngIf="notification.targetType" class="header-column-value" data-cy="notification-target-type">
+                <img src="assets/img/logo-{{notification.targetType}}.png" alt="{{notification.targetType}} logo" />
+                {{ notification.TargetTypeLabels[notification.targetType] }}
+              </td>
+              <td class="header-column-value" data-cy="notification-rule-type">{{ notification.AlertTypeLabels[notification.ruleType] }}</td>
+              <td class="header-column-value" data-cy="notification-target-url">{{ notification.targetUrl }}</td>
+            </tr>
+          </tbody>
+        </table>
+        <chef-tab-selector [value]="tabValue">
+          <chef-option value='details'>Details</chef-option>
+        </chef-tab-selector>
+      </chef-page-header>
+      <section class="page-body">
+        <form [formGroup]="updateForm">
+          <chef-form-field>
+            <label for="name">Name <span aria-hidden="true">*</span></label>
+            <input chefInput id="name" formControlName="name" type="text" [resetOrigin]="saveSuccessful" autocomplete="off" data-cy="name-input"/>
+            <chef-error *ngIf="(nameCtrl.hasError('required') || nameCtrl.hasError('pattern')) && nameCtrl.dirty">
+              Name is required.
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field>
+            <label>
+              <span class="label">Webhook Type <span aria-hidden="true">*</span></span>
+              <chef-select
+                (change)="changeSelectionForWebhookType($event)"
+                class="notification-select-menu"
+                [value]="notification.targetType">
+                <chef-option *ngFor="let key of targetKeys" [value]="key">
+                  <img src="assets/img/logo-{{key}}.png" alt="{{key}} logo" />
+                  {{ notification.TargetTypeLabels[key] }}
+                </chef-option>
+              </chef-select>
+            </label>
+          </chef-form-field>
+          
+          <chef-form-field>
+            <label>
+              <span class="label">Failure Type  <span aria-hidden="true">*</span></span>
+              <chef-select
+                (change)="setFailureType($event)"
+                class="notification-select-menu"
+                [value]="notification.ruleType">
+                <chef-option *ngFor="let key of alertTypeKeys" [value]="key">
+                  {{ notification.AlertTypeLabels[key] }}
+                </chef-option>
+              </chef-select>
+              <div *ngIf="displayCriticalControlsCheckbox()">
```

Comment:
One more function in the template here that should be resolved

Reply:
fixed.